cubedash/_stac.py

```diff
@@ -1012,6 +1012,71 @@ def collections():
     )
 
 
+@bp.route("/queryables")
+def queryables():
+    """
+    Define what terms are available for use when writing filter expressions for the entire catalog
+    Part of basic CQL2 conformance for stac-api filter implementation.
+    See: https://github.com/stac-api-extensions/filter#queryables
+    """
+    return _utils.as_json(
+        {
+            "$schema": "http://json-schema.org/draft-07/schema#",
+            "$id": "https://example.org/queryables",
```

Comment:
The `$id` should be the current URL, not `example.org`. 

Although the response document is hard coded, in future it may change to include details on the queryable fields, and I'd expect that to be different per explorer instance.